### PR TITLE
Update to stable version of  sqflite_sqlcipher

### PR DESCRIPTION
**Pull request by davidmartos96**

### CODE REVIEW

File: extras/encryption/pubspec.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ environment:
   sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
-  moor: ^4.0.0-nullsafety.0
-  sqflite_sqlcipher: ^2.0.0-nullsafety
+  moor: '>=4.0.0 <5.0.0'
+  sqflite_sqlcipher: '>=2.0.0 <3.0.0'
```
Comment (simolus3) [inline extras/encryption/pubspec.yaml:11]: I think those are equivalent to the following which IMO are easier to read

```suggestion
  moor: ^4.0.0
  sqflite_sqlcipher: ^2.0.0
```

**Review by simolus3 — COMMENTED** [review]
Thanks! Let's just keep using the carret syntax here.

File: extras/encryption/pubspec.yaml (hunk 1 above)
Comment (davidmartos96) [reply extras/encryption/pubspec.yaml:11]: Ok, sounds good!

**Comment by simolus3** [comment]
Thanks!

**Pull request merged**